sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -494,6 +496,10 @@ class tf32;
 |precision::tf32 | float | The TF32 type has a 19 bit format with one
 sign bit, 8 exponent bits (offering the same range as float), and 10
 mantissa bits (offering the same precision as sycl::half).
+|precision::sint4 | int32_t | Eight 4-bit signed integers are packed
+into 32 bits.
+|precision::uint4 | int32_t | Eight 4-bit unsigned integers are packed
+into 32 bits.
```

Comment:
How will the `sint4` and `uint4` abstract types work for the "intel" version of `joint_matrix_apply`, which provides the "row" and "col" for each element?  Since each abstract type holds 4 matrix elements, which "row" and "col" will you provide?  Maybe the callback should be called once for each matrix element, rather than once for each storage element?